test/integration_test/Dockerfile

```diff
@@ -1,4 +1,8 @@
-FROM ubuntu
+FROM golang:1.11.2 
```

Comment:
Not a big deal, but in future you can consider bumping the golang version to something higher:

https://hub.docker.com/_/golang?tab=tags